addon/components/o-s-s/power-select.ts

```diff
@@ -63,18 +69,7 @@ export default class OSSPowerSelect extends BaseDropdown<OSSPowerSelectArgs> {
       return;
     }
 
-    scheduleOnce('afterRender', this, () => {
```

Comment:
Move to private method to prevent the following warning:
![Screenshot 2025-05-06 at 15 24 25](https://github.com/user-attachments/assets/bc2acb73-4da9-4499-bb06-35e079fd56d8)